components/Ledger/LedgerDialog.tsx

```diff
@@ -411,22 +411,26 @@ const LedgerDialog = ({
                 setIsLoading(true);
                 const createAccount = await Ledger.createAccount(accountAddress);
```

Comment:
I see that now it will still try to create an account if the account doesn't exist. Does it mean that the address can have tokens in it but the account still doesn't exist?

Reply:
Yes, if it is this is the first time logging in and address is not recognised on the chain, the user will be asked to create an account and pay small tx fee. If address has no tokens in it, the user will be unable to create an account. I've tried sending some CELO tokens to new address I was trying to create an account with using ceclocli but it always failed. 
I've read this on CELO official website: `To be able to use the wallet and create an account, a user must be invited through their phone number by a verified user. [...] The invitation fee allows for invitees to be able to pay for verification fees and complete the verification process upon redemption of the invitation code.`
So users should have no problems creating an account as long as they have been invited to the network.

Reply:
That means even if that address holds tokens, an account can't be created if it's not verified with a phone number?